features/step_definitions/main_structure_steps.rb

```diff
@@ -94,3 +94,8 @@
 Then (/^I wait for form to load/) do
   wait_for_element_exists("* id:'nav_pane'")
 end
+
+Then (/^I robust touch the "([^\"]*)" text$/) do |text|
+  tap_when_element_exists("* {text CONTAINS[c] '#{text}'}")
+  step("I wait for 10 seconds")
```

Comment:
I found that sometimes when a click caused lag calabash would expect the next element to appear before the device had caught up. This just encapsulates that rather than waiting manually